stubs/setuptools/setuptools/command/build_ext.pyi

```diff
@@ -1,14 +1,16 @@
 from distutils.command.build_ext import build_ext as _build_ext
 from typing import Any
 
+from setuptools import Command
+
 have_rtld: bool
 use_stubs: bool
 libtype: str
 
 def if_dl(s): ...
 def get_abi3_suffix(): ...
 
-class build_ext(_build_ext):
+class build_ext(_build_ext, Command):
```

Comment:
This might need to be type ignored.

Reply:
Done as of How does this work exactly? At runtime I just see class build_ext(_build_ext):; is there some monkeypatching going on?.